build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0-alpha3'
+        classpath 'com.android.tools.build:gradle:2.0.0-beta6'
```

Comment:
I assume any project in production must use stable version or, at least, the beta Gradle plugin as you should have android studio alfa version to use it which usually has some bugs.
so by using alfa plugin you force people to use android studio alfa with no real reason.